File: BRaVe-Mobile-Backend/Models/Activity_Management/Kit.cs
using System.Text.Json.Serialization;

namespace BRaVe_Mobile_Backend.Models
{
    public class Kit
    {
        public int DistributionId { get; set; }
        public int KitId { get; set; }
        [JsonPropertyName("sku")]
        public string SKU { get; set; }
        public string ExternalId { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string Source { get; set; }
        public int Quantity { get; set; }
        public string? TargetType { get; set; } = "K";
        public List<DistrItem> items { get; set; }

    }
}

File: BRaVeMobile/app/src/main/java/intl/iom/bravemobile/models/distributions/Kit.java
package intl.iom.bravemobile.models.distributions;

import java.util.List;

public class Kit {

    public String kitId;
    public String sku;
    public String externalId;
    public String title;
    public String description;
    public String source = "I";
    public String targetType = "K";
    public int quantity;
    public List<DistItem> items;

}
